src/modules/dataTypes.js

```diff
@@ -1,15 +1,26 @@
 import i18n from '../locales/index.js'
+import {
+    VALUE_TYPE_BOOLEAN,
+    VALUE_TYPE_TRUE_ONLY,
+    VALUE_TYPE_INTEGER,
+    VALUE_TYPE_INTEGER_POSITIVE,
+    VALUE_TYPE_INTEGER_NEGATIVE,
+    VALUE_TYPE_INTEGER_ZERO_OR_POSITIVE,
+    VALUE_TYPE_NUMBER,
+    VALUE_TYPE_UNIT_INTERVAL,
+    VALUE_TYPE_PERCENTAGE,
+} from './valueTypes.js'
 
 export const CHART_AGGREGATE_AGGREGATABLE_TYPES = [
```

Comment:
@janhenrikoverland  This isn't used anywhere in analytics and is not exported. Should it be removed?

Reply:
It used to be used here https://github.com/dhis2/analytics/pull/787/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#diff-84194648b47c3b8ebd22f1db71fb0e6465aeb596863b7b51635f88b628823c80L564, but since https://github.com/dhis2/analytics/pull/787 it's just been commented out. So looks safe to remove IMO

Reply:
![image](https://user-images.githubusercontent.com/12590483/166678495-dbbf19d5-6700-4307-bb33-a23c1f065a67.png)
attaching a screenshot if the link doesn't work